txhelpers/swap.go

```diff
@@ -262,7 +266,7 @@ func TxAtomicSwapsInfo(tx *chainjson.TxRawResult, outputSpenders map[uint32]*Out
 	// Check if any of this tx's outputs are contracts. Requires the output to
 	// be spent AND the spending input to have the correct sigscript type.
 	for _, vout := range tx.Vout {
-		if vout.ScriptPubKey.Type != txscript.ScriptHashTy.String() {
+		if vout.ScriptPubKey.Type != stdscript.STScriptHash.String() {
```

Comment:
Another pervasive problem, using the TxRawResult's strings instead of decoding the tx bytes and determining script types.  Much yet to do in this PR.  I likely won't wrap this up and move this out of draft until the dcrwallet PR is merged, and blockchain/v4, but I will be slowly resolving these legacy issues.